test/gpu/native/assertOnFailToGpuize.1.good

```diff
@@ -0,0 +1,3 @@
+warning: The prototype GPU support implies --no-checks. This may impact debuggability. To suppress this warning, compile with --no-checks explicitly
+assertOnFailToGpuize.chpl:23: error: Loop containing assertOnGpu() is not Gpuizable
```

Comment:
Is Gpuizable user-facing terminology we're already established (down to whether the proper spelling is "Gpuizable", "GPUizable", or "GPU-izable", or something else?)

Reply:
Oh jeez. That's a good question (something I'd like to answer for our tech report and slides before I end up using this a hundred times).

As a team we (at least as far as I know) don't really have any documentation standards or best practices when it comes to coining new terms like this.

Spelled in all lower case letters like "Gpuizable" seems pretty unrecognizable if you're not already familiar with the term. So if we want to have it as a user facing term I'd say we should say "GPUizable". We'd still use "gpuizable" and "Gpuizable" in function and variable names.

For the purposes of deferring having to make a decision now I think I'll update the error messages to say something like "Loop containing assertOnGpu() is not eligible to execute on a GPU" but it's something we may want to think about before I update the tech report and do release slides.

Reply:
I actually prefer "not eligible to run on GPU" to the original one, personally. 👍

Reply:
I've updated the wording as can be seen in this change: f9d2392ad4669a82556ee57a34d325fadb50d2a4